packages/zostso/__tests__/__system__/cli/issue/cli.zos-tso.issue.cmd.system.test.ts

```diff
@@ -24,6 +30,9 @@ let acc: string;
 
 describe("zos-tso issue command", () => {
 
+    const jsonObject = yaml.safeLoad(fs.readFileSync(propfile, "utf8"));
+    acc = jsonObject.tso.account;
```

Comment:
Since running tests conditionally based on the system properties may be a more frequent practice in the future. I say we need to build some of this logic into a `test_utilities` class/group to make tests more maintainable in the future (if this is the path we want to move forward with)